src/components/Button/Button.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import "./Button.scss";
+
+function Button(props) {
+  const { color, text, className } = props;
```

Comment:
```suggestion
const Button = ({ color, text, className }) => {
```
tham khảo cách này nhé

Reply:
cách này nên dùng cho những component có props đúng không anh, hay là component nào cũng dùng cách này